src/libaktualizr/uptane/directorrepository.cc

```diff
@@ -32,12 +28,10 @@ bool DirectorRepository::targetsSanityCheck() {
       if (ecu_ids.find(ecu.first) == ecu_ids.end()) {
         ecu_ids.insert(ecu.first);
       } else {
-        last_exception = Uptane::InvalidMetadata(type.toString(), Role::Targets().ToString(), "Found repeated ECU ID.");
-        return false;
+        throw Uptane::InvalidMetadata(type.toString(), Role::TARGETS, "Found repeated ECU ID.");
```

Comment:
It would be ideal if we could list the repeated ID in question if that doesn't make the tuf-test-vector integration too challenging.

Reply:
Or in a LOG_ERROR?

Reply:
Yeah, that's probably good enough. :)